README.md

```diff
@@ -1,24 +1,152 @@
+## :hammer_and_wrench: Built With <a name="built-with"></a>
+### Tech Stack <a name="tech-stack"></a>
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li>Ruby on Rails</li>
+  </ul>
+</details>
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li>Ruby on Rails</li>
+  </ul>
+</details>
+<details>
+<summary>Database</summary>
+  <ul>
+    <li>Postgresql</li>
+  </ul>
+</details>
 
-* Database creation
+<!-- Features -->
 
-* Database initialization
+### Key Features <a name="key-features"></a>
+- We can add, remove, update and display recipe.
+- It has user authentication.
 
-* How to run the test suite
 
-* Services (job queues, cache servers, search engines, etc.)
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-* Deployment instructions
+<!-- GETTING STARTED -->
 
-* ...
+## :computer: Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+In order to run this project you need:
+```
+ - Laptop/Desktop.
+ - Internet.
+ - Ruby.
+ - Rails.
+ - Vs Code.
+```
+### Setup
+Clone this repository to your desired folder:
+```
+ - mkdir my-folder
+ - cd my-folder
+ -https://github.com/Ade179/Budget-App.git
+ - cd Recipe-app
+ - bundle install
+ - rails db:create
+ - rails db:migrate
+```
+### Usage
+
+To run the project, execute the following command:
+- rails server
+
+### Run tests
+
+- rspec.
+
+### Check Linter
+
+- rubocop
+- npx stylelint "**/*.{css,scss}"
+
+### Deployment
+
+- Netlify
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **AdeJuwon DanieL Adesanya**
+
+- GitHub: [@githubhandle](https://github.com/Ade179)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/adejuwon-adesanya-237b54239/)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## :telescope: Future Features <a name="future-features"></a>
+- **Enhance design.**
+- **Connect to the Database.**
+- **Create Responsive Interface.**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## :handshake: Contributing <a name="contributing"></a>
+Contributions, issues, and feature requests are welcome!
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## :star:️ Show your support <a name="support"></a>
+If you like this project you can fork it.
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## :pray: Acknowledgments <a name="acknowledgements"></a>
+I would like to thank... <a href="https://www.microverse.org/?gclid=CjwKCAiArY2fBhB9EiwAWqHK6s-2-x4d57Pghz47XT1BgsYuF81ZprM-k-IwzI0_L96nV0SQ93A8ExoCVnQQAvD_BwE" title="planet icons">Microvers for giving me this opportunity to build this project</a>
```

Comment:
- [ ] The README should contain an "Acknowledgements" section or similar that gives appropriate credit to [Gregoire Vella on Behance](https://www.behance.net/gregoirevella) the [author of the original design](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=), as required by the [Creative Commons license of the design](https://creativecommons.org/licenses/by-nc/4.0/).

Reply:
Done. Thank you